proyecto prueba/Processes/NewCustomer.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using proyecto_prueba.Entities;
+
+namespace proyecto_prueba.Processes
+{
+    public class NewCustomer
+    {
+        public Customers NewCustomerMetodo()
```

Comment:
there could more clear if you make this changes.
public class Customers
 +    {
 +        public Customers Create()//or Register